README.md

```diff
@@ -214,49 +122,49 @@ All the javascript dependencies are listed in `package.json`. To install them ru
 
 
 
-##### Set up the database and start redis
+### 5. Set up the database and start redis
 
-  1. Install [postgres](https://www.postgresql.org/download/) if you don't have
-     it already. If you're using a package manager, you need to make sure you install
-     the following packages: `postgresql`, `postgresql-contrib`, and `postgresql-server-dev-all`
-     which will be required to build `psycopg2` python driver.
+Install [postgres](https://www.postgresql.org/download/) if you don't have it already. If you're using a package manager, you need to make sure you install the following packages: `postgresql`, `postgresql-contrib`, and `postgresql-server-dev-all` which will be required to build `psycopg2` python driver.
 
-  2. Make sure postgres is running
+Make sure postgres is running
 
-         service postgresql start
-         # or pg_ctl -D /usr/local/var/postgresql@9.6 start
+    service postgresql start
+    # or pg_ctl -D /usr/local/var/postgresql@9.6 start
 
-  3. Create a database user with username `learningequality` and password `kolibri`:
+Create a database user with username `learningequality` and password `kolibri`:
 
-         sudo su postgres
-         psql
-         # mac: psql postgres
-           CREATE USER learningequality with NOSUPERUSER INHERIT NOCREATEROLE CREATEDB LOGIN NOREPLICATION NOBYPASSRLS PASSWORD 'kolibri';
+    sudo su postgres
+    psql
+    # mac: psql postgres
+      CREATE USER learningequality with NOSUPERUSER INHERIT NOCREATEROLE CREATEDB LOGIN NOREPLICATION NOBYPASSRLS PASSWORD 'kolibri';
 
-  4. Create a database called `gonano`
+Create a database called `gonano`
 
-         CREATE DATABASE "gonano" WITH TEMPLATE = template0 OWNER = "learningequality";
+    CREATE DATABASE "gonano" WITH TEMPLATE = template0 OWNER = "learningequality";
 
 
 
-##### Run all database migrations and load constants
+### 6. Run all database migrations and load constants
 
-You'll only need to run these commands once, to setup the necessary tables and
-constants in the database:
+You'll only need to run these commands once, to setup the necessary tables and constants in the database:
```

Comment:
Actually, these may need to be run at various times after pulling updates from git. Maybe just say "These commands setup the necessary tables and contents in the database." for now?